core/domain/exp_domain.py

```diff
@@ -3323,41 +3323,50 @@ def _convert_states_v20_dict_to_v21_dict(cls, states_dict):
         for state_dict in states_dict.values():
             state_dict['content']['html'] = (
                 html_cleaner.convert_to_text_angular(
-                    state_dict['content']['html']))
+                    html_cleaner.tag_content_to_textangular(
```

Comment:
Why are we repeating this construct everywhere? We should just subsume it within convert_to_text_angular().

Reply:
Do you suggest moving the call to ``tag_content_to_textangular`` inside ``convert_to_text_angular``?